source/clear-linux/get-started/virtual-machine-install/virtualbox-cl-installer.rst

```diff
@@ -1,258 +1,383 @@
-   *Empty* under the *Controller: IDE* from.
+#. From :guilabel:`Storage Devices` in middle column, click the blue
+   disk labeled *Empty*.
 
-#. From the right-hand *Attributes* column, click the blue CD disk next to 
-   the *Optical Drive* drop down menu and click *Choose Virtual Optical 
+#. From the :guilabel:`Attributes` menu, click the blue CD disk next to
+   the *Optical Drive* drop down menu and click *Choose Virtual Optical
    Disk File...*
 
-   .. image:: ./figures/vbox/vbox-vm-settings-mount-ISO.png
-      :alt: Mounting an ISO in VirtualBox VM Settings  
+   .. figure:: figures/vbox/virtualbox-cl-installer-05.png
+      :scale: 100%
+      :alt: Choose Virtual Optical Disk Drive
 
-#. A *choose a virtual optical disk file* browser window will appear. 
-   Navigate to the extracted ISO file, select it, and click *Open*.
+      Figure 5: Choose Virtual Optical Disk Drive
 
-   .. image:: ./figures/vbox/vbox-vm-settings-browse-ISO.png
-      :alt: Mounting an ISO in VirtualBox VM Settings  
+#. Where there appears :guilabel:`Please choose a virtual optical disk file`,
+   select the ISO file and click *Open*.
 
-#. Click *OK* to exit the *VM Settings* menu and return to the main 
-   |VBM|.
+   .. figure:: figures/vbox/virtualbox-cl-installer-06.png
+      :scale: 100%
+      :alt: Mounting an ISO
 
+      Figure 6: Mounting an ISO
 
+#. Select :guilabel:`OK` to exit and return to the main |VBM|.
 
-Install |CL| using the installer
-================================
+Install |CL| with live-server installer
+=======================================
 
-#. Start the VM from the |VBM| by selecting the |CL| VM and 
-   clicking *Start*
+#. In the |VBM|, select virtual machine you created and click *Start*.
 
-   .. image:: ./figures/vbox/vbox-start-vm.png
-      :alt: Starting a VirtualBox VM
+   .. figure:: figures/vbox/virtualbox-cl-installer-07.png
+      :scale: 100%
+      :alt: Start the installer
 
-#. A new window of the VM console will appear and boot into the |CL|
-   installer. Follow the steps in :ref:`bare-metal-install-desktop` to
-   install |CL| onto the VM virtual disk.
+      Figure 7: Start the installer
 
-.. note::
-   Do not choose a different kernel from the installer. **kernel-lts**, the 
-   Long Term Support (LTS) kernel is required for |VB| driver compatibility.
+   .. note::
 
+      To release the mouse cursor from the VM console window, press the right
+      :kbd:`Ctrl` key on the keyboard.
 
-#. After |CL| installation is complete, the VM will reboot and return to the 
-   |CL| installer. 
+#. When :guilabel:`Clear Linux Installer` in boot manager appears,
+   select :kbd:`Enter`. Do not install the bundle `desktop-autostart`.root
 
-.. note::
-   To release the mouse cursor from the VM console window, press the right Ctrl key on the keyboard.
+#. Follow the steps in :ref:`bare-metal-install-server` to
+   install |CL| onto the VM virtual disk. Note:
```

Comment:
@mvincerx , did you mean to put this `Note:` on a new line?

Reply:
No, I meant it as is. Thanks.